### PR TITLE
URL Fixes for fs_mark

### PR DESCRIPTION
**Pull request by narasimhan-v**
* Changed the URL to a working one.
* Moved the Readme section in yaml file to README file.
* Removed unwanted data from yaml file.

Signed-off-by: Narasimhan V <sim@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by narasimhan-v** [comment]
```
# avocado run fs_mark.py -m fs_mark.py.data/fs_mark.yaml 
JOB ID     : 0ee00678ca0ee66e3a30a5295c6e71ad6cc6c9e2
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2017-09-17T12.59-0ee0067/job.log
 (1/3) fs_mark.py:fs_mark.test;ext4-d834: PASS (40.59 s)
 (2/3) fs_mark.py:fs_mark.test;xfs-3b06: PASS (13.01 s)
 (3/3) fs_mark.py:fs_mark.test;btrfs-7654: PASS (77.66 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 144.81 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2017-09-17T12.59-0ee0067/results.html
```

File: io/disk/fs_mark.py.data/README.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+About fs_mark
+-------------
+
+fs_mark options:
+ -n 10000000: The number of files to be tested per thread (more on that later)
+ -s 400; Each file will be 400KB
+ -L 1: Loop the testing once (fs_mark testing)
+ -S 0: Issue no sync() or fsync() during the creation of the file system. 
+ -D 10000: There are 10,000 subdirectories under the main root directory
+ -d /mnt: The root directory for testing; for this particular test, we are using only 1  #   root directory
```
Comment (PraveenPenguin) [inline io/disk/fs_mark.py.data/README.txt]: please wrap this line in 80 char length

File: io/disk/fs_mark.py.data/README.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ -N 1000: 1,000 files are allocated per directory
+ -t 10: Use 10 threads for building the file system
+ -k: Keep the file system after testing
+
+SAMPLE RUN
+  A typical run of the program would look like this:
+    ./fs_mark -d /mnt -s 10240 -n 1000
+
+
```
Comment (PraveenPenguin) [inline io/disk/fs_mark.py.data/README.txt]: this blank line can be removed

File: io/disk/fs_mark.py.data/README.txt [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Multiplexer Input Parameters
+----------------------------
+
+disk		- disk on which the test is to be run
+dir		- dir on which the disk needs to be mounted and run
+num_files	- number of files allocated per directory
+size		- size of each file
+
```
Comment (PraveenPenguin) [inline io/disk/fs_mark.py.data/README.txt]: here as well

**Comment by narasimhan-v** [comment]
@PraveenPenguin Thanks for the review. Please have a look now.

**Pull request merged**